src/components/navbar.tsx

```diff
@@ -3,63 +3,73 @@ import { Link } from 'react-router-dom';
-                <Link to="/account">
-                  <img
-                    src={user.image}
-                    width="20"
-                    height="20"
-                    className="border-circle"
-                    alt={`${user.firstName} ${user.lastName}`}
-                  />
+          <div>
+            <Link to="/" className="navLogo">
+              skräp
+            </Link>
+          </div>
+          <div className="navList">
+            <ul id="nav-mobile" className="right hide-on-small-only">
+              {/* <li>
+                <Button>
+                  <Alert />
+                </Button>
+              </li> */}
+              <li>
+                <Link to="/jobs">Jobs</Link>
+              </li>
+              <li>
+                <Link to="/map">Map</Link>
+              </li>
+              <li>
+                <Link to="/inbox" className={user.clearance ? '' : 'ghost'}>
+                  Inbox
+                </Link>
+              </li>
+              <li>
+                <Link to="/account" className={user.clearance ? 'ghost' : ''}>
+                  Account
                 </Link>
-                <div className="user-profile-subnav green accent-4">
-                  <button
-                    className="btn btn-small"
-                    onClick={() => dispatch(logoutThunk())}
-                    type="button"
-                  >
-                    Logout
-                  </button>
-                  <Link to={`/stripe/${user.id}`}>Stripe</Link>
-                </div>
               </li>
-            )}
-            <Button color="primary">
-              <Alert />
-            </Button>
-            <li>
-              <Link to="/jobs">Jobs</Link>
-            </li>
-            <li>
-              <Link to="/map">Map</Link>
-            </li>
-            <li>
-              <Link to="/inbox" className={user.clearance ? '' : 'ghost'}>
-                Inbox
-              </Link>
-            </li>
-            <li>
-              <Link to="/account" className={user.clearance ? 'ghost' : ''}>
-                Account
-              </Link>
-            </li>
-          </ul>
+              {!!user.clearance && (
+                <li className="user-profile">
+                  <Link to="/account">
+                    <img
+                      src={user.image}
+                      width="20"
+                      height="20"
+                      className="border-circle"
+                      alt={`${user.firstName} ${user.lastName}`}
+                    />
+                  </Link>
+                  <div className="user-profile-subnav">
+                    <div>
+                      <button
+                        className="nav-btn"
+                        onClick={() => dispatch(logoutThunk())}
+                        type="button"
+                      >
+                        Logout
+                      </button>
```

Comment:
Can we replace with UI component? https://github.com/SkrapLitter/capstone/pull/85/files#diff-2190f7797b70783925014d04daa36acdR155-R164

Reply:
the way this looks on the page is just like any other link in the navbar.
it's weird, i know, but if we try an onClick with an a tag, eslint is going to yell at us. that's why i did it this way.

Reply:
hm. weird. didn't have this problems. anyway i have approved